fixtures/fixtures.js

```diff
@@ -51,7 +51,7 @@ var Fixtures = {
 Fixtures.apply = function (mock) {
     //TODO: example regex route;
     //TODO: example route-parsed route;
-    mock.onGet('/traders/1/vouchers').reply(200,this.traderVouchers["1"]);
+    mock.onGet('/traders/1/vouchers').reply(200, this.traderVouchers["1"]);
```

Comment:
1 in this cas eis a label key. it happens to be the zero'th element in that array. I was toying with installing a route-parsing plugin to get laravel style route param niceness. It hasn't happened yet, but the naming system stuck.